server/data/incidentTypeConfiguration/types.ts

```diff
@@ -27,6 +27,8 @@ export interface QuestionConfiguration {
 }
 
 export interface AnswerConfiguration {
+  /** Answer identifier */
```

Comment:
I think we ought to say that this is internal and only to be used for distinguishing and linking form fields. And that this value won't be sent to the api nor downstream data consumers

Reply:
Aren't ids always internal? 🤔 I guess this is not going to be sent to the API, or at least not at the moment.

I'll try to think of some wording that makes sense.

Reply:
quite often you’d use an “id” to talk with an api so i just wanna make sure we remember that we won’t in this case :)